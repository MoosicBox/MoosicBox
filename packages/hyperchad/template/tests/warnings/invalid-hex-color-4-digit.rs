use hyperchad_template::container;

fn main() {
    container! {
        div color=#1a2b { "4-digit hex - invalid" }
    };
}
