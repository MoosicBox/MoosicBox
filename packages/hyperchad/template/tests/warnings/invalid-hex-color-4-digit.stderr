error: Invalid hex color '#1a2b'. Hex colors must be 3, 6, or 8 hexadecimal digits (0-9, a-f)
 --> tests/warnings/invalid-hex-color-4-digit.rs:5:19
  |
5 |         div color=#1a2b { "4-digit hex - invalid" }
  |                   ^^^^^
